README.md

```diff
@@ -1,341 +1,26 @@
 # Thunder
-A C++ platform abstraction layer for generic functionality.
 
-It can be built successfully on both Linux and Windows. First, make sure you have the dependencies sorted out, and then go to the proper OS section:
-* [Linux Build](#Linux)
-* [Windows Build](#Windows)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) 
 
--------------------------------------------------------------------------------------------
-## Documentation
+![Linux Build](https://github.com/rdkcentral/Thunder/actions/workflows/Build%20Thunder%20on%20Linux.yml/badge.svg) ![Windows Build](https://github.com/rdkcentral/Thunder/actions/workflows/Build%20Thunder%20on%20Windows.yml/badge.svg) ![Unit Test](https://github.com/rdkcentral/Thunder/actions/workflows/Test%20Thunder.yml/badge.svg)
 
-Documentation on Thunder can be found here: [Documentation](doc/README.md)
 
--------------------------------------------------------------------------------------------
-## Dependencies
-After JsonGenerator.py and StubGenerator.py were modified to run with the Python 3.5 version, some additional actions might be required. However, no additional actions are necessary while using Buildroot or Yocto. When running these scripts manually or on Windows, make sure that Python 3.5 or higher is used:
+Thunder (also known as WPEFramework) is an open-source plugin-based device abstraction layer, where business functionality can be implemented as plugins and applications can query and control those plugins. Using Thunder provides a consistent interface-driven development model for both plugins and client applications, with an RPC engine that is suited to both web-based and native apps.
 
-```Shell
-python --version
-```
+Designed from the ground up for embedded platforms and written in C++11, Thunder can be run on even the most low-power of devices (including ARM and MIPS-based platforms).
 
-You must also satisfy the requirement of having the **jsonref** library installed. This can be accomplished with the following command:
+# Documentation
 
-```Shell
-pip install jsonref
-```
+All documentation and build instructions for Thunder can be found here: [Documentation](https://rdkcentral.github.io/Thunder/)
 
--------------------------------------------------------------------------------------------
-**Internal plugins**
-* [Controller](Source/WPEFramework/doc/ControllerPlugin.md) - activates and deactivates the configured plugins.
+# Copyright and License
 
--------------------------------------------------------------------------------------------
-<a name="Linux"></a>
-## Linux (Desktop) Build
-These instructions should work on Raspberry PI or any Linux distribution.
+If not stated otherwise in this file or this component's LICENSE file the following copyright and licenses apply:
```

Comment:
Rather than borrowing from the code headers, please can this (lines 18-26) be reworded to simply say:
Thunder is Copyright 2018 Metrological and licensed under the Apache License, Version 2.0.  See the LICENSE and NOTICE files in the top level directory for further details.

Reply:
Done